src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java

```diff
@@ -49,7 +49,8 @@
 import cz.matejcik.openwig.Engine;
 import cz.matejcik.openwig.formats.CartridgeFile;
 import locus.api.objects.extra.Location;
-import locus.api.objects.extra.Waypoint;
+
+//import locus.api.objects.extra.Waypoint;
```

Comment:
From history:
`locus.api.objects.extra.Waypoint` was refactored to `locus.api.objects.extra.Point`, then integrated in `locus.api.objects.geoData.Point`.